Inventory view (#1062)
percona/pmm-doc: docs/details/dashboards/dashboard-inventory.md

# PMM Inventory

The **Inventory** dashboard is a high-level overview of all objects  registered in PMM.

To check your inventory list, go to <i class="uil uil-cog"></i> **Configuration** > **Inventory**.

![!image](../../_images/Inventory.png)

Inventory objects form a hierarchy with Node at the top, then Service and Agents assigned to a Node.

This information is detailed in the two tabs available on this page:

**Services** tab

> Shows individual services, the nodes on which they run, and the Agents against which they are assigned.

Reviewer: ```suggestion
Shows individual services, the nodes on which they run, and the Agents that help collect the service metrics.
```

The **Status** column shows the status of your databases based on metrics coming directly from the database. 
The **Monitoring** column summarizes the status of all the Agents assigned to the service.

> You can check Query Analytics information and the Service Overview Dashboard for each service by clicking on the ![!image](../../_images/dots-three-vertical.ico) in the **Options** column.

Reviewer: Those icons don't render well, they break the layout (

From here you can also check additional information about the service, by clicking the ![!image](../../_images/arrow-downward.ico). This expands the service entry to show reference information like service labels and IDs.

> Each instance of a service gets a `service_id` value that is related to a `node_id`. Examples are MySQL, Amazon Aurora MySQL. This feature also allows to support multiple mysqld instances on a single node, with different service names, e.g. `mysql1-3306`, and `mysql1-3307`.

Reviewer: ```suggestion
Each instance of a service gets a `service_type` attribute so one can clearly tell what type of database it is, for instance: `mysql`, `postgresql`, `mongodb`, etc. Every service is related to a certain node via its `node_id` attribute. This feature allows to support multiple instances on a single node, with different service names, e.g. `mysql1-3306`, and `mysql1-3307`.
```

> Each binary (exporter, agent) running on a client will get an `agent_id` value. Examples:

Reviewer: ```suggestion
Each binary (exporter, agent) running on a client will get an `agent_type` value. Examples:
```

> - `pmm-agent` one is the top of the tree, assigned to a `node_id`

Reviewer: ```suggestion
- `pmm-agent` is at the top of the tree, assigned to PMM Agent itself
```

> - `node_exporter` is assigned to pmm-agent `agent_id`

Reviewer: ```suggestion
- `node_exporter` is assigned to an agent that extracts the node metrics
```

> - `mysqld_exporter` and QAN MySQL Perfschema are assigned to a `service_id`.

Reviewer: ```suggestion
- `mysqld_exporter` and `qan-mysql-perfschema-agent` are assigned to agents that extract metrics from mysql and its performance schema respectively.
```
